VitruvioHost/Plugins/Vitruvio/Source/Vitruvio/Public/VitruvioComponent.h

```diff
@@ -235,8 +239,9 @@ class VITRUVIO_API UVitruvioComponent : public UActorComponent
 	int32 RandomSeed;
 
 	/** CGA Reports from generation. */
-	FReportArray Reports;
-	
+	UPROPERTY(EditAnywhere, DisplayName = "Reports", Category = "Vitruvio")
```

Comment:
I think we should make this "VisibleAnywhere" otherwise you can add reports into the map from the Details panel.